natgeo/mortar
# [SYTO-741] Remove left border from pull quote attribution title

These changes remove the left border from the pull quote attribution title so that the border doesn't visually appear twice when its paired with the attribution name

There are also a few other clean up changes requested by Kevin sprinkled in.

To test:

- Pull bugfixes/SYTO-741
- Run` npm test` to verify linting
- Run `gulp serve`
- navigate to http://localhost:3000/typography/
- Verify attribution title in pull quote no longer has a double border
- Spot check headings section to verify layout not negatively impacted by changes
